SpatialGDK/Source/SpatialGDKServices/Private/Interop/Connection/EditorWorkerController.cpp

```diff
@@ -10,6 +10,14 @@ namespace
 {
 struct EditorWorkerController
 {
+	// Only issue the worker replace request if there's a chance the load balancer hasn't acknowledged
+	// that the previous session's workers have disconnected. There's no hard `heartbeat` time for this as
+	// it's dependent on multiple factors (fabric load etc.), so this value was landed on after significant
+	// trial and error.
+	const int64 WorkerReplaceThresholdSeconds = 8;
+
+	const FString ServicePort = TEXT("9876");
```

Comment:
Magic number? Could this be in a constants file?

Reply:
Discussed offline, since they are only used here we're not gonna move them for now.